spec/integ/postgres.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { PostgresStore } from "../../src";
+import { getPgDatabase, initPostgres, isPostgresTestingEnabled } from "../helpers/postgres-helper";
+
+
+// Only run the tests if we've enabled postgres testing.
+let descr = isPostgresTestingEnabled() ? describe : xdescribe;
+
+class TestPostgresStore extends PostgresStore { }
+
+descr('PostgresStore', () => {
+    let store: TestPostgresStore|undefined;
+    beforeAll(() => {
+        initPostgres();
+    })
+
+    it('can construct a simple database from schema', async () => {
+        store = new TestPostgresStore([], {
+            url: await getPgDatabase(),
+        });
+        await store.ensureSchema();
+    });
+
```

Comment:
This would benefit from a test that runs schema upgrades.  Here's a simple one that I've confirmed passes:
```suggestion
    it('can run schema upgrades', async () => {
        store = new TestPostgresStore([
            sql => sql.begin(s => [
                s`CREATE TABLE v1_users (mxid TEXT UNIQUE NOT NULL);`,
            ]),
            sql => sql.begin(s => [
                s`CREATE TABLE v2_rooms (mxid TEXT UNIQUE NOT NULL);`,
            ]),
        ], {
            autocreateSchemaTable: true,
            url: await getPgDatabase(),
        });
        await store.ensureSchema();
    });

```

Reply:
That would be helpful :grin: . Thanks!